etcd/server/service.sls

```diff
@@ -1,13 +1,48 @@
 {%- from "etcd/map.jinja" import server with context %}
 {%- if server.enabled %}
 
+# get binaries and install packages
 etcd_packages:
   pkg.installed:
-  - names: {{ server.pkgs }}
-{%- if server.get('engine', 'systemd') %}
-  - require:
-    - file: /etc/default/etcd
-{%- endif %}
+    - names: {{ server.pkgs }}
+
+/tmp/etcd/:
```

Comment:
Copying from images is done by othere deployment tools like Kargo. 

I would prefer this way since it is used by salt-formula-kubernetes as well - so we reuse the same approach.